README.md

```diff
@@ -114,101 +178,262 @@ another desired outcome. If a policy already exists for your host, you can skip
       role: !host <NAME_OF_HOST>
       privileges: [read, execute]
       resource: !variable <NAME_OF_VARIABLE>
-```
-
-_NOTE:_ If you cannot remember the admin's API key and you have the proper privileges, you can run `conjurctl role retrieve-key cucumber:user:admin` from the Conjur server to retrieve the admin's API key and load the policy
-
-3. Load the policy into Conjur using the CLI like so: `conjur policy load <PATH_TO_POLICY> <NAME_OF_POLICY>.yml`
-
-_NOTE:_ For this policy, `<PATH_TO_POLICY>` is root. Meaning, this policy has been loaded into the root node of the policy tree. You may want to change this. For more information about the Conjur policy tree, check out [this](https://www.conjur.org/blog/understanding-conjur-policy/) blog post.
-
-4. Once the policy is loaded (for the first time), an ID and API key will be returned for the host. It should look something like:
-```bash
+   ```
+
+   _NOTE:_ If you are running Conjur OSS, cannot remember the admin's API key, and you
+   have the proper privileges, you can run `conjurctl role retrieve-key <CONJUR_ACCOUNT>:user:admin`
+   from the Conjur server container to retrieve the admin's API key.
+
+3. Load the policy into Conjur using the CLI:
+   ```sh-session
+   $ conjur policy load <PATH_TO_POLICY> <NAME_OF_POLICY>.yml
+   ```
+
+   _NOTE:_ For this policy, `<PATH_TO_POLICY>` is root. Meaning, this policy has been
+   loaded into the root node of the policy tree. You may want to change this depending
+   on your policy tree. For more information about the Conjur policy tree, check out
+   [this](https://www.conjur.org/blog/understanding-conjur-policy/) blog post.
+
+4. Once the policy is loaded (for the first time), an ID and API key will be returned
+   for the host. It should look something like:
+   ```sh-session
+$ conjur policy load root policy.yml
 Loaded policy 'root'
 {
   "created_roles": {
-    "cucumber:host:<POLICY_ID>/<NAME_OF_HOST>": {
-      "id": "cucumber:host:<POLICY_ID>/<NAME_OF_HOST>",
+    "myorg:host:<POLICY_ID>/<NAME_OF_HOST>": {
+      "id": "myorg:host:<POLICY_ID>/<NAME_OF_HOST>",
       "api_key": "3scdjaz3aq90vn26avdv11s2mf2h2wnwfzw3cbgbja1jspzyc1rdzwqx"
     }
   },
   "version": 1
 }
```

Comment:
the content in the code block needs to be indented to render properly

Reply:
No longer relevant - example scrapped